src/org/mozilla/javascript/SecurityUtilities.java

```diff
@@ -6,68 +6,79 @@
 
 package org.mozilla.javascript;
 
-import java.security.AccessController;
-import java.security.PrivilegedAction;
 import java.security.ProtectionDomain;
 
-/**
- * @author Attila Szegedi
- */
-public class SecurityUtilities
-{
+/** @author Attila Szegedi */
+public abstract class SecurityUtilities {
+    static final SecurityBridge bridge = makeBridge();
+
+    private static SecurityBridge makeBridge() {
+
+        String[] classNames = {
+            "org.mozilla.javascript.SecurityBridge_custom",
+            // Check, if SecurityManager class exists
+            // TODO: Shoud we check JDK version here?
```

Comment:
It is not known when the actual APIs are going to be removed, so I would not check the Java version.